APP_README.md

```diff
@@ -0,0 +1,91 @@
+﻿# Your Project Name
+
+TODO: Give a **brief description** of the app.
+Explain the **high level goals** and the **main** features.
+
+TODO: Make sure you adjust or augment the information that is provided by default in this document.
+e.g. If you decide to change the solution structure, you should mention it in the [Solution Structure section](#solution-structure).
+
+## Project Template
+
+This repository was generated using the **nventive Mobile Template**.
+- Version: [{{app-template-version}}](https://www.nuget.org/packages/NV.Templates.Mobile/{{app-template-version}})
+- Commit: [{{app-template-commit-short-sha}}](https://github.com/nventive/UnoApplicationTemplate/tree/{{app-template-commit-full-sha}})
+- Date: {{app-template-commit-date}}
+
+[**View the template changes since the generation of this project**.](https://github.com/nventive/UnoApplicationTemplate/compare/{{app-template-commit-full-sha}}..main)
+
+## Environment and Prerequisites
+
+### Local Development Requirements
+All development is expected to be done from Visual Studio in a Windows environment.
+
+- .Net 7
```

Comment:
This is not relevant to flutter

Reply:
Yes this is normal I won't be updating the whole documentation it will be out of scope.

Reply:
Hum, in that case, that should be empty with a TODO then to avoid confusing devs

Reply:
Yes but IMO it's much easier to leave it as is because we will have to copy from the Uno Application Template anyway in the future to make it slightly different.